deps/wazuh_testing/wazuh_testing/scripts/qa_ctl.py

```diff
@@ -169,7 +169,8 @@ def validate_parameters(parameters):
     if parameters.run_test:
         for test in parameters.run_test:
             tests_path = os.path.join(WAZUH_QA_FILES, 'tests')
-            if 'test exists' not in local_actions.run_local_command_with_output(f"qa-docs -e {test} -I {tests_path}"):
+            if 'Documentation block ok' not in \
```

Comment:
Remove all changes of this file

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/2074/commits/42f76f5d1c6aafa4f5ebab6e94e7eab97e7c8954